what4/src/What4/Utils/Process.hs

```diff
@@ -59,7 +60,8 @@ withProcessHandles path args mcwd action = do
           { std_in  = CreatePipe
           , std_out = CreatePipe
           , std_err = CreatePipe
-          , create_group = True
+          , create_group = False
+          , delegate_ctlc = True
           , cwd = mcwd
```

Comment:
I just noticed that these options are duplicated in the SMTLib2 module - would it be worth consolidating them to make sure they get updated together since this seems to be fragile? I'm not sure that the extra effort is really worth it, just a thought.

Reply:
Yup, and also in the `Yices` module, where I forgot to update them. That suggests it is worth consolidating.